build.rs

```diff
@@ -0,0 +1,5 @@
+extern crate skeptic;
+
+fn main() {
+    skeptic::generate_doc_tests(&["README.md"]);
```

Comment:
Do you want to know how I know you use OS X? Case-insensitive file system! The filename is `Readme.md`. You can actually just rename the file if you want to, several other projects I have use upper-case `README`.